sisl/viz/plotly/_presets.py

```diff
@@ -0,0 +1,37 @@
+__all__ = ["add_presets", "get_preset"]
+
+PRESETS = {
+
+    "dark": {
+        "layout": {"template": "sisl_dark"},
+        "bands_color": "#ccc",
+        "bands_width": 2
+    },
+
+}
+
+
+def add_presets(**presets):
+    """
+    Registers new presets
+
+    Parameters
+    ----------
+    **presets:
+        as many as you want. Each preset is a dict.
+    """
+
```

Comment:
remove empty lines

Reply:
Which ones? :sweat_smile:

Reply:
You have lots of empty lines between documentation and single lines of code, small methods. It isn't strictly necessary to change now, but it would be nice to streamline to the rest of the code base. :)